tests/testthat/_snaps/error-call.md

```diff
@@ -173,3 +173,54 @@
       x These names are invalid:
         * "..." at location 1.
 
+# subscript validation reports correct error calls
+
+    Code
+      (expect_error(my_function()))
+    Output
+      <error/rlang_error>
+      Error in `vctrs::num_as_location()`:
+      ! `missing` must be one of "propagate" or "error".
```

Comment:
So it knows to report the vctrs function here rather than "in `my_function()`"? I guess that is right?

Reply:
yes this is the correct error call. See https://github.com/r-lib/vctrs/pull/1507#issuecomment-1022272169 for the implementation.